### DISCUSSION TITLE
Standard naming for new modules

### DISCUSSION THREAD

**Discussion by jmdegeorge** [CVWRSM/cvwrsm#99] [General]
As the devops process moves forward, we'd like to make sure we standardize the naming of new modules at the top level.
Using the hyphen for naming conventions and short names for the module folders has been a common practice. 
I would like to suggest the following module folder names for the next modules in the queue for the DevOps updates:

\wrims-core (already complete)
\wrims-plugin     <= wrims_v2\wrimsv2_plugin
\dwr-hecdssvue  <= wrims_v2\gov.ca.dwr.hecdssvue
\dwr-jdiagram     <= wrims_v2\gov.ca.dwr.jdiagram

NOTE: The final output jar does not have to match the module folder name, but it is generally a good practice for it to do so. 

Thanks in advance for your feedback!
~Josh

**Comment by hxiedwr** [comment]
Try to have:
\wrims-plugin <= wrims_v2\gov.ca.dwr.wrims
